src/client/interpreter/configuration/interpreterSelector.ts

```diff
@@ -138,12 +147,21 @@ export class InterpreterSelector implements IInterpreterSelector {
             workspaceFolder.uri
         );
     }
-    private async getWorkspaceToSetPythonPath(): Promise<WorkspacePythonPath | undefined> {
+    private async getConfigTarget(): Promise<
+        | {
+              folderUri: Resource;
+              configTarget: ConfigurationTarget;
+          }
+        | undefined
+    > {
         if (
             !Array.isArray(this.workspaceService.workspaceFolders) ||
             this.workspaceService.workspaceFolders.length === 0
         ) {
-            return undefined;
+            return {
+                folderUri: undefined,
+                configTarget: ConfigurationTarget.Global
+            };
```

Comment:
```suggestion
            ({
                folderUri: undefined,
                configTarget: ConfigurationTarget.Global
            })
```

I think you don't need a return there. Putting it in parenthesis should work. see https://stackoverflow.com/a/47755280